frontend/packages/pipelines-plugin/src/components/pipelines/detail-page-tabs/pipeline-details/PipelineVisualizationTask.scss

```diff
@@ -15,6 +15,8 @@ $border-color: var(--pf-global--BorderColor--light-100);
     cursor: pointer;
   }
   &--icon-spin {
+    filter: blur(0);
+    -webkit-filter: blur(0);
```

Comment:
Added this to avoid the wobbly status icon while rotation.
Before fix:
![wobbling_small](https://user-images.githubusercontent.com/9964343/121316402-ec941500-c926-11eb-8fc8-d5e38539d6dd.gif)
After fix the rotation is smooth:
![small-without-wobbling](https://user-images.githubusercontent.com/9964343/121316408-ee5dd880-c926-11eb-8fba-3430f2199c01.gif)